test/conformance/kuttl/kuttl-test.yaml

```diff
@@ -1,6 +1,8 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestSuite
 testDirs:
+  # Autogen tests
+- ./test/conformance/kuttl/autogen/should-autogen
```

Comment:
Other autogen tests directories are not added here, any reason ?

Reply:
Good catch, they are probably not run :(

Reply:
Fixed

Reply:
cool

Reply:
Good catch.